.github/workflows/tools/set-pr-status

```diff
@@ -11,7 +11,19 @@ function api_call {
 INPUT_PR="$1"
 OSNAME="$2"
 STATE="$3"
-DESCRIPTION="$4"
+
+if [ "$STATE" = pending ]
+then DESCRIPTION="git-annex building ..."
+elif [ "$STATE" = success ]
+then DESCRIPTION="git-annex built successfully"
+elif [ "$STATE" = failure ]
+then DESCRIPTION="git-annex build failed!"
+elif [ "$STATE" = cancelled ]
```

Comment:
Is "canceled" the same as it would be if skipped?  In report to this PR I see some canceled but overall status is "1 failing, 8 successful, 3 skipped, and 6 in progress checks", so I would assume that "canceled" -> "skipped", just want to confirm.

Reply:
I believe "cancelled" means that either a human hit the "Cancel workflow" button or that the maximum runtime was exceeded.  "Skipped" is a thing that is done to jobs & steps rather than a status; you would never have a status of "skipped."

Reply:
Ok, let's proceed then, thank you!